tests/building_extend_test.py

```diff
@@ -228,3 +230,40 @@ def test_from_dict():
     assert new_bldg.properties.energy.construction_set.name == \
         'Thermal Mass Construction Set'
     assert new_bldg.to_dict() == bd
+
+
+def test_to_dict_revit():
+    """Test the to_dict method in a way that mimics how the export from Revit happens."""
+    pts_1 = (Point3D(0, 0, 0), Point3D(0, 10, 0), Point3D(10, 10, 0), Point3D(10, 0, 0))
+    pts_2 = (Point3D(10, 0, 0), Point3D(10, 10, 0), Point3D(20, 10, 0), Point3D(20, 0, 0))
+    pts_3 = (Point3D(0, 10, 0), Point3D(0, 20, 0), Point3D(10, 20, 0), Point3D(10, 10, 0))
+    pts_4 = (Point3D(10, 10, 0), Point3D(10, 20, 0), Point3D(20, 20, 0), Point3D(20, 10, 0))
+    pts_5 = (Point3D(0, 0, 3), Point3D(0, 10, 3), Point3D(10, 10, 3), Point3D(10, 0, 3))
+    pts_6 = (Point3D(10, 0, 3), Point3D(10, 10, 3), Point3D(20, 10, 3), Point3D(20, 0, 3))
+    pts_7 = (Point3D(0, 0, 6), Point3D(0, 10, 6), Point3D(10, 10, 6), Point3D(10, 0, 6))
+    room2d_1 = Room2D('Office 1', Face3D(pts_1), 3)
+    room2d_2 = Room2D('Office 2', Face3D(pts_2), 3)
+    room2d_3 = Room2D('Office 3', Face3D(pts_3), 3)
+    room2d_4 = Room2D('Office 4', Face3D(pts_4), 3)
+    room2d_5 = Room2D('Office 5', Face3D(pts_5), 3)
+    room2d_6 = Room2D('Office 6', Face3D(pts_6), 3)
+    room2d_7 = Room2D('Office 7', Face3D(pts_7), 3)
+    story_1 = Story('Office Floor 1', [room2d_1, room2d_2, room2d_3, room2d_4])
+    story_2 = Story('Office Floor 2', [room2d_5, room2d_6])
+    story_3 = Story('Office Floor 3', [room2d_7])
+    story_1.solve_room_2d_adjacency(0.01)
+    story_2.solve_room_2d_adjacency(0.01)
+    story_1.set_outdoor_window_parameters(SimpleWindowRatio(0.3))
+    story_2.set_outdoor_window_parameters(SimpleWindowRatio(0.35))
+    story_3.set_outdoor_window_parameters(SimpleWindowRatio(0.6))
+    building = Building('Office Building', [story_1, story_2, story_3])
+    building.auto_assign_top_bottom_floors()
+
+    building.to_dict()
+    """
+    f_dir = 'C:/Users/chris/Ladybug Tools Google Drive/laybug_tools_llc/basecamp/' \
+        'pollination-revit/schema-samples'
+    dest_file = f_dir + '/3_building_schema_sample.json'
+    with open(dest_file, 'w') as fp:
+        json.dump(building.to_dict(True), fp, indent=4)
+    """
```

Comment:
What you suggest there is a good long-term solution but I think that I would rather keep these as they are right now just because it makes it easier to locate them among the other tests when I need to write out new JSON samples for other people. Maybe an even better solution long-term is just to pull this code that is meant to generate samples out of the tests and give them their own folder or repo.

Reply:
Let me think about it since I have code like this over several repos and, when I make the change, I should implement it across all of the tests.

Reply:
I think what you are suggesting is correct and they should not be in tests. You should add a module / command as a utility to generate these. And having done the same I would say start fixing it from here and then if you get a chance fix the older ones. Otherwise it might got so big that you will never fix it.